### DISCUSSION TITLE
docker-compose.yml VS docker-compose-hub.yml?

### DISCUSSION THREAD

**Discussion by mohaa7** [makeplane/plane#2442] [Q&A]
What's the difference between [docker-compose-hub.yml](https://github.com/makeplane/plane/blob/master/docker-compose-hub.yml) and [docker-compose.yml](https://github.com/makeplane/plane/blob/master/docker-compose.yml)? (Based on the [documentation](https://docs.plane.so/self-hosting), Plane is run by `docker-compose-hub.yaml`)

**Comment by devSparkle** [comment]
[docker-compose.yml](https://github.com/makeplane/plane/blob/master/docker-compose.yml) is setup to build the docker images from the source code of the repository. It requires you to download the entire repo.

[docker-compose-hub.yml](https://github.com/makeplane/plane/blob/master/docker-compose-hub.yml) uses the pre-built *Docker Hub* (hence the name) images; which means you can use it standalone, without having to retrieve the entire repo and build the images yourself.

**Reply by ovizii** [reply]
Thanks for clarifying, maybe you should mention this here: https://docs.plane.so/self-hosting as not many people want to clone a repo, keep it updated and build their own images.
